### PR TITLE
Add log bench

### PR DESCRIPTION
**Pull request by alex391a**
* Add log bench in tensile_host.cpp

* Update tensile input in runContractionProblem (c api)

* Remove  original log bench

### CODE REVIEW

File: library/src/amd_detail/rocblaslt/src/tensile_host.cpp [hunk 1]
```diff
@@ -333,6 +333,169 @@ namespace
...
+            problem.useGradient() ? "--gradient" : "",
+            problem.useE() ? "--use_e" : "",
+            problem.useBias() ? "--bias_vector" : "",
+            problem.useBias() ? "--bias_source" : "",
+            problem.useBias() ? problem.tensor(problem.biasSrc()).getName() : "",
+            GEN_BENCH_ARG(hipDataType_to_bench_string, "--a_type", tensile2HipType(problem.a().dataType())),
+            GEN_BENCH_ARG(hipDataType_to_bench_string, "--b_type", tensile2HipType(problem.b().dataType())),
+            GEN_BENCH_ARG(hipDataType_to_bench_string, "--c_type", tensile2HipType(problem.c().dataType())),
+            GEN_BENCH_ARG(hipDataType_to_bench_string, "--d_type", tensile2HipType(problem.d().dataType())),
+            GEN_BENCH_ARG(hipDataType_to_bench_string, "--compute_type", tensile2HipType(problem.computeType())),
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/tensile_host.cpp]: compute_type translation is more complicated than this formula.

File: library/src/amd_detail/rocblaslt/src/rocblaslt_mat.cpp [hunk 2]
```diff
@@ -200,68 +200,7 @@ rocblaslt_status rocblaslt_matmul_impl(const rocblaslt_handle       handle,
...
-            scaleD ? "--scaleD" : "",
-            scaleAlphaVec ? "--scaleAlpha_vector" : "",
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--a_type", type_a),
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--b_type", type_b),
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--c_type", type_c),
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--d_type", type_d),
-            GEN_BENCH_ARG(rocblaslt_compute_type_to_bench_string, "--compute_type", compute_type),
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--scale_type", scale_type),
-            GEN_BENCH_ARG(hipDataType_to_bench_string, "--bias_type", bias_type),
-            rocblaslt_epilogue_to_bench_string(epilogue));
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/rocblaslt_mat.cpp:261]: remove unused function

File: library/src/amd_detail/rocblaslt/src/tensile_host.cpp [hunk 3]
```diff
@@ -333,6 +333,197 @@ namespace
             maxWorkspaceBytes);
     }
 
+    const char* tensileComputeInputType_to_bench_string(Tensile::DataType typeCompute)
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/tensile_host.cpp]: please try some f8 problem type to check if this function work normally.

File: library/src/amd_detail/rocblaslt/src/tensile_host.cpp [hunk 4]
```diff
@@ -333,6 +333,229 @@ namespace
...
+    const char* tensileComputeInputType_to_bench_string(Tensile::DataType typeCompute,
+                                                        Tensile::DataType F32XdlMathOp,
+                                                        Tensile::DataType typeComputeInput,
+                                                        Tensile::DataType typeA,
+                                                        Tensile::DataType typeB)
+    {
+        switch(typeCompute)
+        {
+        case  Tensile::DataType::Float:
+            break;
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/tensile_host.cpp:345]: Have you tried fp32 input/fp32 output problem?

**Review by jichangjichang — APPROVED** [review]
lgtm

**Comment by jichangjichang** [comment]
Since you didn't squash your commit manually, please remember to use "squash merge" but NOT "Rebase and merge"

**Review by AndySu12 — APPROVED** [review]
LGTM

**Pull request merged**